ui/app/styles/components/popup-menu.scss

```diff
@@ -8,7 +8,7 @@
     box-shadow: $box-shadow, $box-shadow-middle;
     padding: 0;
     position: relative;
-    width: 200px;
+    width: 210px;
```

Comment:
This component is used a lot. Just want to make sure it's okay to change this globally.

Reply:
I spot checked a couple other pop-up menus and it didn't seem to be an issue. But worth noting!